### ISSUE TITLE
There is no clean way to update a single app

### ISSUE THREAD

**Issue by pedro823** [vtex/toolbelt#492]
TL;DR: There is no option to update a single app to its latest version.

## The Problem
`vtex update` updates all apps, every time.
`vtex install <app>` Doesn't work, says it is already installed.

The way to do it, today:

- choose a random version that doesn't exist.
- `vtex install <app>@<version>`. Toolbelt will answer "Hey, i didn't find this version of the app. Here's the whole version list"
- Find the latest version inside that huge list.
- `vtex install <app>@<correct-version>`.

## Possible Solution
Make `vtex update` receive optional arguments, and they should be the __only__ apps to update.
